config/env_spec_request.go

```diff
@@ -350,11 +352,11 @@ func (e *EnvironmentSpecRequest) JWTAuthentications() []*JWTAuthentication {
 }
 
 // looks up the JWTAuthentication by name and runs verification
-// returns true if found and verified
+// returns nil if found and verified
```

Comment:
nit: returning nil on success is assumed with Go; maybe rewrite this (and others) as "return error if not found, or not verified"

Reply:
Done